css/main.css

```diff
@@ -32,7 +29,7 @@ body {
 }
 
 footer {
-  margin: 200px 0;
+  margin: 0px 0px 30px;
```

Comment:
Why not just say 
`margin-bottom: 30px;`

Reply:
- When i added space Like: `.status { ` but prettier removed the space.
- Can you please tell me why there is need to add spacing between css selectors and the curly braces?
Thanks for reviewing my code.

Reply:
Oh, it's just for better readability/cleaner look, nothing else actually